src/child/actions/sidebar.js

```diff
@@ -1,80 +1,73 @@
-export const SEARCH_CLICKED = 'SEARCH_CLICKED';
-export const SEARCH_STARTED = 'SEARCH_STARTED';
-export const SEARCH_INPUT = 'SEARCH_INPUT';
-export const SEARCH_ERROR = 'SEARCH_ERROR';
-export const CLEAR_SEARCH = 'CLEAR_SEARCH';
-export const SEARCH_FINISHED = 'SEARCH_FINISHED';
-export const FAV_CLICKED = 'FAV_CLICKED';
-export const TOGGLE_FAVOURITE = 'TOGGLE_FAVOURITE';
-export const SELECTION = 'SELECTION';
-export const UNSELECT = 'UNSELECT';
+
+import ACTIONS from '../enums/actions.js';
```

Comment:
Not sure, but if the constants are to be stored in nested objects, then I'd be tempted to do the following, so to not import constants which aren't required here and allows you to be a bit more terse when using them (e.g. `ACTIONS.SIDEBAR.SEARCH_INPUT` vs `ACTIONS.SEARCH_INPUT`).

```js
import { SIDEBAR as ACTION_TYPES } from '../enums/actions.js';

//...

ACTION_TYPES.SOME_ACTION
```